src/main/app/src/components/common/CookieModal.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState } from 'react';
+
+import { makeStyles, Checkbox } from '@material-ui/core';
+import ArrowDropDown from '@material-ui/icons/ArrowDropDown';
+import ArrowDropUp from '@material-ui/icons/ArrowDropUp';
+import Cookies from 'js-cookie';
+import Markdown from 'markdown-to-jsx';
+
+import { useContentful } from '#/src/hooks';
+
+const useStyles = makeStyles({
+  modalBackdrop: {
+    width: '100%',
+    height: '100%',
+    position: 'fixed',
+    'z-index': '9999',
+    left: '0',
+    top: '0',
+    'background-color': 'rgba(0, 0, 0, 0.5)',
+    color: '#000000',
+    display: 'block',
+    'overflow-y': 'auto',
+  },
+
+  modal_header: {
+    'margin-top': '0px',
+    'margin-left': '3%',
+    'margin-right': '3%',
+  },
+
+  icon: {
+    position: 'absolute',
+  },
+
+  modalText: {
+    'margin-top': '15px',
+    'margin-bottom': '15px',
+    'margin-left': '3%',
+    'margin-right': '3%',
+    'white-space': 'pre-wrap !important',
+  },
+
+  modalContent: {
+    position: 'absolute',
+    'z-index': '9999',
+    'background-color': 'white',
+    width: '60%',
+    border: '1px solid #ccc',
+    'box-shadow': '1px 1px 1px black',
+    padding: '16px',
+    left: '20%',
+    top: '5%',
+    'box-sizing': 'border-box',
+    'border-radius': '10px',
+    a: {
+      color: '#0a789c !important',
+      'margin-top': '5px',
+      'margin-bottom': '15px',
+      'text-decoration': 'none',
+    },
+    '@media (max-width:960px)': {
+      width: '90%',
+      left: '5%',
+    },
+  },
+
+  settings: {
+    'margin-top': '15px',
+    'margin-left': '2%',
+  },
+
+  settingsCheckbox: {
+    outline: '1px #3A7A10',
+    'box-shadow': 'none',
+    'font-size': '2em',
+    ':checked': {
+      'background-color': '#3A7A10',
+      icon: 'pointer',
+    },
+  },
+
+  settingsButton: {
+    'margin-top': '15px',
+    'margin-right': '5px',
+    height: '40px',
+    color: '#3A7A10',
+    'background-color': 'white',
+    padding: '4px 12px 4px',
+    cursor: 'pointer',
+    border: '2px solid transparent',
+    'border-color': '#3A7A10',
+    'border-radius': '4px',
+    'font-weight': '600',
+    float: 'right',
+    '&:hover': {
+      backgroundColor: '#e6ffc1',
+    },
+  },
+
+  acceptButton: {
+    'margin-top': '15px',
+    'margin-right': '5px',
+    height: '40px',
+    color: '#FFFFFF',
+    'background-color': '#3A7A10',
+    padding: '4px 12px 4px',
+    cursor: 'pointer',
+    border: '1px solid transparent',
+    'border-radius': '4px',
+    'border-color': '#3A7A10',
+    'font-weight': '600',
+    float: 'right',
+    '&:hover': {
+      backgroundColor: '#43a047',
+    },
+  },
+
+  textExpandLink: {
+    color: '#3A7A10',
+    'margin-top': '5px',
+    'margin-bottom': '15px',
+    'margin-left': '3%',
+    'font-weight': 'bold',
+  },
+});
+
```

Comment:
Nyt on todella paljon kovakoodattuja värejä jotka eivät ota huomioon teemaa. Ei hajauteta tyylejä komponenttikohtaisesksi, koska muuten teemavärejä joutuu ylläpitään monessa paikkaa.

Näistä pitäisi nyt vaihtaa värit käyttään näitä: src\colors.ts

Toisekseen jos komponentille on määritelty näin paljon tyyliä niin ei käytetä valmiita komponentteja. Pitäisi käydä läpi mitä näistä voi jättää kokonaan pois. Ainakin buttonit pitäisi käyttää suoraan valmiina.